extensions/adobe/experience/adcloud/searchads/campaign.schema.json

```diff
@@ -0,0 +1,166 @@
+            "Ended",
+            "Pending",
+            "Inactive",
+            "Disapproved",
+            "Incomplete",
+            "Offline",
+            "On Hold (Editing)",
+            "On Hold (Other)",
+            "Duplicate",
+            "Orphan",
+            "Optimize",
+            "Disabled",
+            "Unknown",
+            "Dummy"
+          ],
+          "meta:enum": {
+            "Active": "Active",
+            "Paused": "Paused",
+            "Deleted": "Deleted",
+            "Ended": "Ended",
+            "Pending": "Pending",
+            "Inactive": "Inactive",
+            "Disapproved": "Disapproved",
+            "Incomplete": "Incomplete",
+            "Offline": "Offline",
+            "On Hold (Editing)": "On Hold (Editing)",
+            "On Hold (Other)": "On Hold (Other)",
+            "Duplicate": "Duplicate",
+            "Orphan": "Orphan",
+            "Optimize": "Optimize",
+            "Disabled": "Disabled",
+            "Unknown": "Unknown",
+            "Dummy": "Dummy"
+          }
+        },
+        "searchads:channelType": {
+          "title": "Channel Type",
+          "type": "string",
+          "description": "Channel Type for the Campaign"
+        },
+        "searchads:budget": {
+          "title": "Budget",
+          "type": "number",
+          "description": "Budget Allocated for the Campaign"
+        },
+        "searchads:budgetType": {
+          "title": "Budget Type",
+          "type": "string",
+          "description": "Budget Type indicates how the allocated budget will be spent",
+          "enum": [
+            "Daily",
+            "Monthly",
+            "Lifetime",
+            "Daily Spend Until Depleted",
+            "Day of Week"
+          ],
+          "meta:enum": {
+            "daily": "Daily",
+            "Monthly": "Monthly",
+            "Lifetime": "Lifetime",
+            "Daily Spend Until Depleted": "Daily spend until depleted",
+            "Day of week": "Day of week"
+          }
+        },
+        "searchads:adDeliveryType": {
+          "title": "Ad Delivery Method",
+          "type": "string",
+          "description": "Ad Delivery method can determine how long your budget lasts",
+          "enum": ["Standard", "Accelerated"],
+          "meta:enum": {
+            "standard": "Standard",
+            "accelerated": "Accelerated"
+          }
+        },
+        "searchads:objective": {
+          "title": "Campaign Objective",
+          "type": "string",
+          "description": "Campaign Objective indicates the primary objective of the campaign - e.g. Promotions, Brands, App Installs.",
+          "enum": ["visit_web", "promote_brand", "install_app", "visit_offer"],
+          "meta:enum": {
+            "visit_web": "visit_web",
+            "promote_brand": "promote_brand",
+            "install_app": "install_app",
+            "visit_offer": "visit_offer"
+          }
+        },
+        "searchads:campaignStartDate": {
```

Comment:
Should we have generic startDate and endDate instead of campaignStartDate and campaignEndDate?

Reply:
Create a generic object startDate and endDate and share it across schemas that is what you meant?  If so, the idea here was to define a complete object in its entirety rather than adding mix-ins or data-types. If that was not the case let me know what your thoughts were